### PR TITLE
Use nx as a task runner

### PR DESCRIPTION
**Pull request by TheLudd**
This sets up nx with shared cache for transpile, test and lint actions. Try
following the new instructions in the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,14 +9,10 @@ To start up a local sandbox environment of the Northlight UI framework:
...
-4. cd into `/tokens`
-5. run `yarn build`
-6. cd into `/icons`
-7. run `yarn build`
-8. cd into `/framework`
-9. run `yarn start`
-
-This will spin up a local environment sandbox at localhost:3008
+4. cd into `/framework`
+5. run `yarn nx demo`
```
Comment (SebCodesTheWeb) [inline README.md:13]: it's enough with yarn demo right?

**Review by SebCodesTheWeb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (TheLudd) [reply README.md:13]: it is not

Comment (TheLudd) [reply README.md:13]: nx runs all dependant builds that is needed since you did the last build

Comment (TheLudd) [reply README.md:13]: You can alternatively install nx globally and run `nx demo`

**Pull request merged**